AutoFlow_OmicsDataHandling/INCA_script_generator.py

```diff
@@ -273,11 +276,17 @@ def add_reactions_to_script(modelReaction_data_I, atomMappingReactions_data_I):
         model_rxn_ids_exp: List of reaction IDs used for the model
 
     """
+    if len(atomMappingReactions_data_I["rxn_id"]) != len(
```

Comment:
If you don't mind adding in the new `biomass_function` parameter to the docstrings that would be great.

Reply:
Perhaps it would also be a good idea to start a separate module with different default biomass functions for the user to choose if it is problematic reading from a .csv file.

Reply:
Hey so I changed it a bit and now it can import the biomass function from the .csv file